### DISCUSSION TITLE
Shaggy's Style and osu!taiko Support

### DISCUSSION THREAD

**Discussion by Saltssaumure** [Saltssaumure/fnf-osu-mania-skin#14] [General]
# What's in this update? #

* Thanks to [Lumyni](https://github.com/flxLumyni) - Added [Shaggy mod](https://gamebanana.com/mods/284121) style centre diamond and arrows.
* Added osu!taiko support with fully animated pippidon.
* Added no swearing alternate files, and removed swears from filenames.
* Minor adjustment to position of HP bar in osu!standard and osu!catch.
* Fixed a few missing sounds and HD textures.

Every download contains all eight Skin.ini files in the alternative-files folder, so if you'd prefer to save space you don't need to download all eight .osks to try out every version.

# Plans for next update #

Now that finally all four modes are supported, fingers crossed this will be the last functional update. All that's left to do is add some cosmetic features:

* Add combobursts (ideas are welcome!)
* Redo osu!standard numbers.

<hr /><em>This discussion was created from the release <a href='https://github.com/Saltssaumure/fnf-osu-mania-skin/releases/tag/v1.6'>Shaggy's Style and osu!taiko Support</a>.</em>

**Comment by LuisTheIdot** [comment]
how does one make the shaggy skin work?

**Reply by felixLumyni** [reply]
- if you don't have the latest version of the skin (or are lazy to do it the other way), **download one of the last 4 .osk files in this release's assets** and open it with osu, it should automatically install and apply it
- otherwise, go into your skin folder and **swap out the skin.ini with one labeled with shaggy in alternate-files**
have fun :)
